core/domain/takeout_service.py

```diff
@@ -83,8 +85,20 @@ def export_data_for_user(user_id):
     for model in models_to_export:
         split_name = re.findall('[A-Z][^A-Z]*', model.__name__)[:-1]
         # Join the split name with underscores and add _data for final name.
+
+        export_data = model.export_data(user_id)
```

Comment:
Can you call this exported_data so that it doesn't start with a verb?

EDIT: Oh interesting, this name seems to already be used. Perhaps exported_model_data?

Reply:
Done.